README.md

```diff
@@ -1,6 +1,40 @@
-# Towards "fearless SIMD"
+<!--
+
+This repo-level readme needs restructuring, pending some Linebender templating decisions.
+https://xi.zulipchat.com/#narrow/channel/419691-linebender/topic/Bikeshedding.20badges/with/452312397
+
+For now, prefer updating the package-level readmes, e.g. fearless_simd/README.md.
+
+-->
+
+<div align="center">
+
+# Fearless SIMD
+
+**Safer and easier SIMD**
+
+[![Linebender Zulip](https://img.shields.io/badge/Linebender-%23vello-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/channel/197075-vello)
+[![dependency status](https://deps.rs/repo/github/linebender/fearless_simd/status.svg)](https://deps.rs/repo/github/linebender/fearless_simd)
+[![Apache 2.0 or MIT license.](https://img.shields.io/badge/license-Apache--2.0_OR_MIT-blue.svg)](#license)
+
+[![Crates.io](https://img.shields.io/crates/v/fearless_simd.svg)](https://crates.io/crates/fearless_simd)
+[![Docs](https://docs.rs/vello/badge.svg)](https://docs.rs/vello)
+[![Build status](https://github.com/linebender/fearless_simd/workflows/CI/badge.svg)](https://github.com/linebender/fearless_simd/actions)
```

Comment:
Nit: For the alt texts here, we should probably follow the pattern from https://github.com/linebender/vello/pull/601

(See also the `{badge1}\[newline]{badge2}` style from that PR, which we should probably also emulate).

But also, we know that we're going to change this some more, so it's fine to land as-is.